learning_journal_basic/setup.py

```diff
@@ -10,17 +10,27 @@
 
 requires = [
     'pyramid',
-    'pyramid_chameleon',
     'pyramid_debugtoolbar',
     'waitress',
+    'pyramid_jinja2',
+    'pyramid_tm',
+    'SQLAlchemy',
+    'transaction',
+    'zope.sqlalchemy',
     ]
 
 tests_require = [
     'WebTest >= 1.3.1',  # py3 compat
     'pytest',  # includes virtualenv
     'pytest-cov',
+    'pytest-watch',
+    'tox',
     ]
 
+development_extra = [
+    'ipython',
+    'pyramid_ipython'
+]
```

Comment:
You add development_extra (good!) but have no entry point to allow you to install these dependencies.